web/static/help/Messaging.md

```diff
@@ -1 +0,0 @@
-../../../doc/help/Messaging.md
```

Comment:
I didn't remove it, i just made a copy of the file. I can remove it but you said duplicate it